src/util.cc

```diff
@@ -2303,8 +2301,7 @@ static void strerror_r_helper(char* result, char* buf, size_t buflen) {
         buf[buflen - 1] = 0;
 }
 
-static void strerror_r_helper(int result, char* buf, size_t buflen) { /* XSI flavor of strerror_r, no-op. */
-}
+static void strerror_r_helper(int result, char* buf, size_t buflen) { /* XSI flavor of strerror_r, no-op. */ }
```

Comment:
That one, the above, and their use is pretty strange :-)

Reply:
Yep, both the current code and the original commit fc33bf2014 are slightly light on details 😉